### PR TITLE
Update mde-device-control-device-installation.md

### PR DESCRIPTION
**Pull request by msbemba**
Appended the note to include  the allow policy won’t take precedence if the Block Removable Storage option is selected in Device Control

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9946

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Prevent installation of removable devices
 This policy setting allows you to specify a list of Plug and Play hardware IDs and compatible IDs for devices that Windows is prevented from installing. By default, this policy setting takes precedence over any other policy setting that allows Windows to install a device.
 
 > [!NOTE]
-> To enable the **Allow installation of devices that match any of these device instance IDs** policy setting to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting.
+> To enable the **Allow installation of devices that match any of these device instance IDs** policy setting to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting. Also, the allow policy won’t take precedence if the Block Removable Storage option is selected in Device Control.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mde-device-control-device-installation.md]:
```suggestion
> To enable the **Allow installation of devices that match any of these device instance IDs** policy setting to supersede this policy setting for applicable devices, enable the **Apply layered order of evaluation for Allow and Prevent device installation policies across all device match criteria** policy setting. Also, the allow policy won’t take precedence if the **Block Removable Storage** option is selected in Device Control.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**